cognee/infrastructure/databases/graph/neptune_analytics_driver/adapter.py

```diff
@@ -526,9 +567,34 @@ async def get_edges(self, node_id: str) -> List[EdgeData]:
         --------
             - List[EdgeData]: A list of EdgeData objects representing edges connected to the node.
         """
-        # TODO: Implement using aws_langchain Neptune Analytics edge retrieval
-        logger.warning(f"Neptune Analytics get_edges method not yet implemented for node: {node_id}")
-        return []
+        try:
+            # Query to get all edges connected to the node (both incoming and outgoing)
+            query = f"""
+            MATCH (n:{self._GRAPH_NODE_LABEL})-[r]-(m:{self._GRAPH_NODE_LABEL})
+            WHERE id(n) = $node_id
```

Comment:
Should we follow neo to inline the id for MATCH clause?
```suggestion
            MATCH (n:{self._GRAPH_NODE_LABEL} {{id=$node_id }})-[r]-(m:{self._GRAPH_NODE_LABEL})
```